jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/RetainableByteBuffer.java

```diff
@@ -38,7 +38,7 @@ public class RetainableByteBuffer extends Retainable.ReferenceCounter
     private final Consumer<RetainableByteBuffer> releaser;
     private final AtomicLong lastUpdate = new AtomicLong(NanoTime.now());
 
-    RetainableByteBuffer(ByteBuffer buffer, Consumer<RetainableByteBuffer> releaser)
+    public RetainableByteBuffer(ByteBuffer buffer, Consumer<RetainableByteBuffer> releaser)
```

Comment:
It doesn't look like that is still needed.

Reply:
reverted